docs/app/install/docker/page.mdx

```diff
@@ -160,6 +160,445 @@ Notice that there is a port number, `8444`, following the hostname in the URLs a
 
 > For more information about how Docker networking works, refer to [Docker Networking Documentation](https://docs.docker.com/network/).
 
+## Persisting Data Across Container Restarts
+
+When running Pelican servers in containers, certain files and directories need to persist across container restarts to maintain server identity, configurations, and data. Without proper volume mounts, restarting a container will cause data loss, including cryptographic keys, databases, and configuration files.
+
+This section describes which data should be persisted for each Pelican server type and how to configure volume mounts appropriately.
+
+### Quick Reference: Essential Volume Mounts
+
+The table below shows the minimum recommended volume mounts for each server type:
+
+| Server Type | Essential Volume Mounts |
+|-------------|------------------------|
+| **All** | `-v /host/pelican/config:/etc/pelican` <br/> `-v /host/pelican/lib:/var/lib/pelican` <br/> `-v /host/pelican/certificates:/etc/pelican/certificates` |
+| **Origin** | Above + <br/> `-v /host/data:/data:ro` (your data directory) |
+| **Cache** | Above + <br/> `-v /host/cache/storage:/var/cache/pelican` (cache storage) |
+| **Director** | Above + <br/> `-v /host/pelican/maxmind:/var/cache/pelican/maxmind` (optional, for GeoIP) |
+| **Registry** | Above (no additional mounts required) |
+
+> **Important**: The `/etc/pelican` directory contains cryptographic keys that establish your server's identity in the federation. Loss of these keys will cause the "namespace is registered with a different key" error for origins.
+
+### Understanding Default Paths
+
+Pelican uses different default paths depending on whether the server runs as root (which is the case in containers):
+- **Root user** (containers): Configuration and data are stored in `/etc/pelican` and `/var/lib/pelican`
+- **Non-root user**: Configuration and data are stored in `~/.config/pelican`
+
+All examples below assume the server runs as root inside a container.
+
+### Data That Should Persist for All Server Types
+
+The following data should be persisted for all Pelican server types (origin, cache, director, registry):
+
+#### Issuer Keys Directory
+
+**Purpose**: Contains PEM-encoded ECDSA private keys used to sign credentials issued by the server. These keys establish the server's cryptographic identity within the federation.
+
+**Default Path**: `/etc/pelican/issuer-keys`
+
+**Volume Mount**:
+```bash
+-v /host/pelican/issuer-keys:/etc/pelican/issuer-keys
+```
+
+**Configuration**: To customize the path, set `IssuerKeysDirectory`:
+```yaml
+IssuerKeysDirectory: /custom/path/to/issuer-keys
+```
+
+**Why it must persist**: If these keys are lost, the server will generate new ones, causing authentication failures with other federation services that trust the old keys. For origins, this is the source of the "namespace is registered with a different key" error.
```

Comment:
Specify that when this error is encountered, the user must contact their federation's administrators to have them delete the old registration. This cannot be fixed without help from the federation admins.